sopel/modules/reddit.py

```diff
@@ -381,8 +388,9 @@ def get_channel_sfw(bot, trigger):
     if not channel:
         channel = trigger.sender
         if channel.is_nick():
-            return bot.say('{}getsfw with no channel param is only permitted in '
-                           'channels'.format(bot.config.core.help_prefix))
+            bot.reply('{}getsfw with no channel param is only permitted in '
+                      'channels'.format(bot.config.core.help_prefix))
```

Comment:
```suggestion
                      'channels.'.format(bot.config.core.help_prefix))
```

Sopel needs to learn proper grammar. 😂